gapis/service/service.proto

```diff
@@ -67,32 +67,45 @@ message ServerInfo {
 
 // Messages that hold a repeated field so they can be used in oneofs.
 
-message Contexts { repeated Context list = 1; }
+message Commands { repeated path.Command list = 1; }
+message Contexts { repeated path.Context list = 1; }
 message Devices { repeated path.Device list = 1; }
-message Hierarchies { repeated Hierarchy list = 1; }
+message Events { repeated Event list = 1; }
 message StringTableInfos { repeated stringtable.Info list = 1; }
-
-message Object {
-  bytes data = 1;
-}
+message Threads { repeated path.Thread list = 1; }
 
 message Value {
```

Comment:
Both of your suggestions also are overloaded - `Any` is used in paths, `Response` isn't great as `Value` is also used by `Set()`.
My justification for the name is when you consider the package each of these are declared in. `pod.Value` is a plain-old-data value, `box.Value` is a boxed value, `service.Value` is any value that is used by the service package. I guess there's a question about whether we could fold `box.Value` and `service.Value` together. I'll think about how easy that would be.

On a side note, this wasn't actually renamed / introduced in these CLs.